src/models/cartsDao.js

```diff
@@ -68,10 +56,36 @@ const getCarts = async (userId) => {
   return product;
 };
 
+const deleteProduct = async (userId, productOptionId) => {
```

Comment:
```suggestion
const deleteCart = async (userId, productOptionId) => {
```
product가 아니라 cart 데이터를 지우고 있네요!